src/app/modules/startup/controller.nim

```diff
@@ -172,10 +172,14 @@ proc init*(self: Controller) =
     self.delegate.onLocalPairingStatusUpdate(self.localPairingStatus)
   self.connectionIds.add(handlerId)
 
-  handlerId = self.events.onWithUUID(SIGNAL_REENCRYPTION_PROCESS_STARTED) do(e: Args):
+  handlerId = self.events.onWithUUID(SignalType.DBReEncryptionStarted.event) do(e: Args):
```

Comment:
Usually, the service is the one that listens to status-go signals like this one, then it sends back a "nim" signal with the appropriate data if needed.
That's to create a separation of concerns I think. It's not a big deal, but I think it would be better to stay consistent.

Reply:
Looks like we don't have a startup service.

There's three controllers breaking the separation of concerns: [startup](https://github.com/status-im/status-desktop/blob/fix/re-encrypt-db/src/app/modules/startup/controller.nim), [wallet activity](https://github.com/status-im/status-desktop/blob/fix/re-encrypt-db/src/app/modules/main/wallet_section/activity/controller.nim#L209) and [node_section](https://github.com/status-im/status-desktop/blob/fix/re-encrypt-db/src/app/modules/main/node_section/controller.nim#L43).

WDYT of creating a follow up task to refactor these modules?

Reply:
Sure. I think the account service might have worked, but it's not a big deal